spec/gutter-spec.coffee

```diff
@@ -0,0 +1,74 @@
+{Gutter, DEFAULT_PRIORITY} = require '../src/gutter'
+
+describe 'Gutter initialization tests.', ->
+  name = 'name'
+  fakeGutterContainer = {}
+
+  it 'Test initializing a Gutter with all options.', ->
```

Comment:
I definitely appreciate the thoroughness, but I don't think this test is really needed. Maybe talking about defaults below would be worth it but even that might be overkill for our general testing level in the codebase.